court/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Court(models.Model):
+    id = models.IntegerField(primary_key=True)
+    x = models.FloatField(default=0)
+    y = models.FloatField(default=0)
+    city = models.CharField(max_length=50, blank=True)
+    neighborhood = models.CharField(max_length=50, blank=True)
+    max_players = models.PositiveIntegerField(default=0)
```

Comment:
Does this mean that if the court is initialized without specifying this field, then it has a max number of zero? Is this the expected behavior?

Reply:
It seems like I was not 100% clear of the attributes, and in second thought, the default attribute is unnecessary. Will be fixed. Thanks.

Reply:
Fixed.